### DISCUSSION TITLE
未知原因导致OCR初始化失败

### DISCUSSION THREAD

**Discussion by shotamiao** [moesnow/March7thAssistant#398] [Q&A]
基于一个群友的反馈
![Screenshot_20241201-160433_ToDesk.jpg](https://github.com/user-attachments/assets/e9336c3b-ccb8-43b2-ac85-43474bfd604a)

![Screenshot_20241201-160958_ToDesk.jpg](https://github.com/user-attachments/assets/1e58c629-b822-4a6a-b9bd-55cef969da01)

期间有让他尝试换其他版本，电脑重启，重新解压或下载，关杀毒，管理员运行，并不能解决

**Comment by TwoWayThinking** [comment]
此问题已解决！需要自己在文件夹March7thAssistant下创建一个名为3rdparty的文件夹，然后将OCR文件解压到此文件夹内，在运行三月七小助手即可正常。
如果还是报错：那就将temp文件夹内也解压一份OCR。
